.travis.yml

```diff
@@ -71,5 +71,6 @@ before_install:
   - phpenv config-rm xdebug.ini
 
 script:
+  - composer stan-setup && composer stan
```

Comment:
* Your file looks a bit outdated, compare it with the given file below.
* https://github.com/spryker-eco/adyen/blob/master/.travis.yml.

Reply:
Done